jwt/api_jwt.py

```diff
@@ -212,7 +212,10 @@ def _validate_iss(self, payload, issuer):
         if "iss" not in payload:
             raise MissingRequiredClaimError("iss")
 
-        if payload["iss"] != issuer:
+        if isinstance(issuer, str):
```

Comment:
do we also need to guard against other invalid values for `issuer`?

see https://github.com/jpadilla/pyjwt/blob/master/jwt/api_jwt.py#L128-L129